### DISCUSSION TITLE
Synergy TV

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#732] [Q&A]
**Channel Name:** Synergy TV
**Channel ID:** SynergyTV.tt
**Country:** Trinidad and Tobago
**Language:** English
**Website:** https://webplayer.tegotv.com/tego-live-tv.php?channel=259
**Stream URL (optional):** N/A
**Notes (optional):** Wikipedia article: https://en.wikipedia.org/wiki/Synergy_TV Note: Like Trinidad and Tobago channels, this is broadcasting from TegoTV service with token.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/tt.m3u#L6-L7
